Gemfile

```diff
@@ -121,9 +121,9 @@ group :default do
   gem 'cancancan'
 
   # Feature flags
-  gem 'flipper', '~> 0.25.0'
-  gem 'flipper-active_record', '~> 0.25.0'
-  gem 'flipper-ui', '~> 0.25.0'
+  gem 'flipper'
+  gem 'flipper-active_record'
+  gem 'flipper-ui'
```

Comment:
Would it be preferable to lock the versions again to the newly identified ones during dependency resolution?  i.e. 1.3.1 for each of these.

I know this can cause dependency lockout in the future, but it stops these randomly upgrading in the future and causing issues because of a regression or because of breaking changes in a version update.